tools/manifestfile.py

```diff
@@ -603,6 +603,9 @@ def main():
         default=os.path.join(os.path.dirname(__file__), "../lib/micropython-lib"),
         help="path to micropython-lib repo",
     )
+    cmd_parser.add_argument(
```

Comment:
Thanks @projectgus I think this is the right approach, but I think we might need the name and path to be both specified explicitly. Suggest we do do `name,path = path.split("=", 1)`

Also the use of `basename` below might be problematic to infer the name:

```
>>> import os
>>> os.path.basename("/foo/bar")
'bar'
>>> os.path.basename("/foo/bar/")
''
```